src/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -58,12 +58,18 @@ Expr InjectPredicate(const Array<Expr>& predicates,
 // Replace data flow appears in all stages given the tensor change.
 // Also update vmap if subsequent dataflow need to be replaced.
 void ReplaceDataFlow(const Array<Stage>& stages,
```

Comment:
add a document why the reverse map is needed. We need to keep an update to the date transitive closure property on the vmap

Reply:
Done